### DISCUSSION TITLE
Kerbalism info integration

### DISCUSSION THREAD

**Discussion by edusaraiva** [yagiziskirik/Kerbal-Telemetry#18] [Ideas]
First of all loved your mod.... top notch dude...
I know you are still developing this... but do you think is possible to integrate info from kerbalism? 
THAT would be huge both for Stock and RP-1 players?
Thanks once again for the great work

**Comment by yagiziskirik** [comment]
Hi @edusaraiva,

I appreciate the support, you are the best! Many of you guys asked me the same thing, so I will also be working on that for the next update! I don't have much spare time right now because things are snowballing for me. But as soon as I can create free time, I will be working on the project again! Thanks for your suggestions!
